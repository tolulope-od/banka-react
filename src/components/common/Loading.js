import React from 'react';

const Loading = () => {
  return (
    <center>
      <img
        src="https://res.cloudinary.com/tolulope-od/image/upload/v1557908999/loading_o1y5v6.gif"
        width="150"
      />
    </center>
  );
};

export default Loading;
