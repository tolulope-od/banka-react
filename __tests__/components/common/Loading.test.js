import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../../../src/components/common/Loading';

describe('<Loading />', () => {
  it('Renders the Loading component', () => {
    const wrapper = shallow(<Loading />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
