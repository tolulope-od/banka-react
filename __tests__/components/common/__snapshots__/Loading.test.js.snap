// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Renders the Loading component 1`] = `
<center>
  <img
    src="https://res.cloudinary.com/tolulope-od/image/upload/v1557908999/loading_o1y5v6.gif"
    width="150"
  />
</center>
`;
